build.gradle

```diff
@@ -98,6 +98,7 @@ subprojects {
     libertyDebug.dependsOn 'libertyStop'    
     libertyStart.dependsOn 'libertyStop', 'test'
     libertyRun.dependsOn   'libertyStop'
+    composeUp.dependsOn ':player-service:addPostgresqlToBuild'
```

Comment:
instead of specifically depending on addPostgresqlToBuild, it would probably be more appropriate if we just depended on `build` or `assemble` in general (I thought we already did) because the .war files need to be built too in order to produce each docker image